README.md

```diff
@@ -35,6 +35,12 @@ Follow the 4-step program:
 3. Add callbacks to prout - ie a GitHub webhook and ideally also a post-deploy hook
 4. Expose the commit id of your build on your deployed site
 
+## Give prout-bot write-access
+
+![Giving prout write-access by going to Settings -> Collaborators and team -> Add people and typing in 'prout-bot'](./artwork/prout-write.gif)
+
+Admin access to the repository is required to give prout-bot write-access. Please contact your team's Engineering Manager for this.
```

Comment:
I think this is good, though in the past I think we've specifically added the 'prout' team, which has one member, https://github.com/prout-bot . Quite often we try to avoid granting access to _specific_ GitHub user accounts, as there's a risk that those permissions won't be removed when the user leaves the organisation, I think?!

In the case of the @prout-bot account, it probably doesn't matter for us.